File: HOL/dotnet/modern-cloud-apps/end/DevCamp.WebApp/Utils/IncidentApiHelper.cs
using IncidentAPI;
using System;
using System.Configuration;

namespace DevCamp.WebApp.Utils
{
    public class IncidentApiHelper
    {
       public static IncidentAPIClient GetIncidentAPIClient()
        {
            var client = new IncidentAPIClient(new Uri(string.Format("https://{0}", ConfigurationManager.AppSettings["INCIDENT_API_URL"])));
            return client;
        }

    }
}

File: HOL/java/07-completed/src/main/java/devCamp/WebApp/Utils/IncidentApiHelper.java
package devCamp.WebApp.Utils;

import devCamp.WebApp.IncidentAPIClient.IncidentAPIClient;

public class IncidentApiHelper {
	public static IncidentAPIClient getIncidentAPIClient() {
		
		String apiurl= System.getenv("INCIDENT_API_URL");
		return new IncidentAPIClient(apiurl);
	}
}
